hr_attendance_report/security/ir.model.access.csv

```diff
@@ -0,0 +1,5 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+hr_attendance_report.access_employee_attendance_report_wizard,access_employee_attendance_report_wizard,hr_attendance_report.model_employee_attendance_report_wizard,hr.group_hr_user,1,0,0,0
+hr_attendance_report.access_custom_excel_class,access_custom_excel_class,hr_attendance_report.model_custom_excel_class,hr.group_hr_user,1,0,0,0
+hr_attendance_report.access_employee_attendance_report_wizard_manager,access_employee_attendance_report_wizard_manager,hr_attendance_report.model_employee_attendance_report_wizard,hr.group_hr_manager,1,1,1,1
```

Comment:
On transient models, the unlink permission is never given. That's reserved to the cleanup cron.